lib/shim/message-shim.js

```diff
@@ -489,6 +440,7 @@ function recordConsume(nodule, properties, spec) {
 
```

Comment:
Since we were all wondering why this function is special... wondering if we should leave a comment at the top of the function saying why it isn't using record()?

Reply:
honestly i didn't study this enough but the more i look at it we may be able to use record.  for now i'll just put a comment explaining why it is not and log a ticket